crxmake.sh

```diff
@@ -59,9 +62,16 @@ sig_len_hex=$(byte_swap $(printf '%08x\n' $(ls -l "$sig" | awk '{print $5}')))
 
 echo "Wrote $crx"
 
+# move crx to artifacts folder in circleci
+if [ $CIRCLE_ARTIFACTS ]; then
```

Comment:
That's great @ericjim this brings us 95% there. Is there a way to automate having a direct link to crx file within the PR page intself (on Github). It's a long short but would it be possible to add a general comment to a PR with a link? Any other way would do. If not this is definitely good already.